packages/vaadin-grid/test/column.test.js

```diff
@@ -552,4 +552,13 @@ describe('column', () => {
       expect(column._bodyTemplate).to.eql(template2);
     });
   });
+
+  it('should not throw an exception when size is changed after removing column', () => {
+    expect(grid.size).to.equal(10);
+    const column = grid.querySelector('vaadin-grid-column');
```

Comment:
Can't the existing `column` reference be used?

Reply:
Done